tracing/axum-logs-endpoint/src/main.rs

```diff
@@ -11,12 +11,11 @@ use shuttle_axum::ShuttleAxum;
 use tracing::info;
 
 mod logger;
-use logger::Logger;
 
 mod state;
 use state::AppState;
 
-#[shuttle_runtime::main(tracing_layer = Logger::make_layer)]
+#[shuttle_runtime::main(tracing_layer = logger::Logger::make_layer)]
```

Comment:
Is this going to work after we merge the logger? I think the custom layer will not be supported anymore.

Reply:
Indeed, the custom tracing layer is no longer supported. Users who want to customize their tracing setup can just opt out of the default now and set it up however they want. I think we can just remove them for now.

Reply:
Should we remove this example then? Or do you mean the `custom-layer` example?

Reply:
Removed the examples in 679f476ec9dd9083269c18c6d9f6885db9962411